### ISSUE TITLE
Finished project page in Sugar wiki

### ISSUE THREAD

**Issue by gen1e** [axk4545/hfoss-sugar-snake#15]
We need to make our own page in the [Sugar Math4Team/RIT/Projects wiki](http://wiki.sugarlabs.org/go/Math4Team/RIT/Projects). I will work on fleshing that out.

**Comment by gen1e** [comment]
Currently waiting on the Sugar Labs team to get back to me about an account.

**Comment by gen1e** [comment]
This is the issue I ran into, where my email got flagged because I wasn't on the mainling list and we have to wait for the list admin to review it. Full text here:

Your mail to 'Systems' with the subject

    Sugar Labs Wiki Account

Is being held until the list moderator can review it for approval.

The reason it is being held:

    Post by non-member to a members-only list

Either the message will get posted to the list, or you will receive
notification of the moderator's decision.  If you would like to cancel
this posting, please visit the following URL: (didn't visit it)

**Comment by ritjoe** [comment]
Just to confirm, your message was sent to this list?

http://lists.sugarlabs.org/archive/systems/2018-April/thread.html

**Comment by walterbender** [comment]
Can you email me with the account id and email address? Not sure why the email did not go through.

**Comment by gen1e** [comment]
Yes I was able to create an account, thanks guys! The email was to "systems@lists.sugarlabs.org" as indicated on wiki login page. I just have to write the page.

**Comment by quozl** [comment]
Thanks.  Perhaps we should also say on the Wiki; "subscribe to the list to maximise your chances of success."  :grin:

Thanks for the game, I've forked your repository to github.com/sugarlabs

**Comment by gen1e** [comment]
@ritjoe We might want to consider adding documentation to our status template when we get the chance, I had to copy and paste from PyCut. But we are on [the RIT projects page](https://wiki.sugarlabs.org/go/Math4Team/RIT/Projects) and [have our own activity page](https://wiki.sugarlabs.org/go/Activities/SugarSnake). So we are good to go! Unless I screwed something up.
